src/components/Common/Buttons/Buttons.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import classes from './Buttons.module.css';
+
+const Buttons = (props) => {
+
+    return (
+        <>
+            {(props.type === "startButton") && <button className={classes.btn} onClick={props.handleClick}>Start</button>}
+            {(props.type === "checkButton") && <button className={classes.btn} onClick={props.checkButtonClickHandler} disabled={!props.enableState} >Check</button>}
```

Comment:
please use only one button and send the label (start, check, ...) as a prop

Reply:
done